src/gc/config.d

```diff
@@ -36,7 +36,7 @@ struct Config
         string s = "GC options are specified as whitespace separated assignments:
     disable:0|1    - start disabled (%d)
     profile:0|1|2  - enable profiling with summary when terminating program (%d)
-    gc:conservative|manual - select gc implementation (default = conservative)
+    gc:conservative|precise|manual - select gc implementation (default = conservative)
```

Comment:
Just a reminder to update the spec too (or at least open up a PR, s.t. it doesn't get lost once this PR has been merged): https://github.com/dlang/dlang.org/blob/master/spec/garbage.dd#L386

Reply:
https://github.com/dlang/dlang.org/pull/2553